fuse_publishers/src/serialized_publisher.cpp

```diff
@@ -64,6 +67,17 @@ void SerializedPublisher::onInit()
   bool latch = false;
   private_node_handle_.getParam("latch", latch);
 
+  ros::Duration graph_throttle_period { 0.0 };
+  double graph_throttle_period_double = graph_throttle_period.toSec();
+  fuse_core::getPositiveParam(private_node_handle_, "graph_throttle_period", graph_throttle_period_double, false);
```

Comment:
Not for this PR, but as long as we have utility parameter loading methods, having one that loads durations would be handy.

Reply:
That would be handy.

I tried to provide one, and also overload `fuse_core::getPositiveParam()` to support `ros::Duration`, but I didn't come with a good solution. I was indeed trying to support `ros::Duration` in the templated `fuse_core::getPositiveParam()` function, but I believe I ran into some issues and I didn't invest more time trying to make it work.

Reply:
Can't we just do this?
```
/**
 * @brief Helper function that loads positive duration values from the parameter server
 *
 * @param[in] node_handle - The node handle used to load the parameter
 * @param[in] parameter_name - The parameter name to load
 * @param[in, out] default_value - A default value to use if the provided parameter name does not exist. As output it
 *                                 has the loaded (or default) value
 * @param[in] strict - Whether to check the loaded value is strictly positive or not, i.e. whether 0 is accepted or not
 */
void getPositiveParam(const ros::NodeHandle& node_handle, const std::string& parameter_name,
                      ros::Duration& default_value, const bool strict = true)
{
  double default_value_sec = default_value.toSec();
  getPositiveParam(node_handle, parameter_name, default_value_sec, strict);
  default_value.fromSec(default_value_sec);
}
```

Reply:
Done, that works if I make it `inline`.

Reply:
You say "Done" but I don't see any new commits. :slightly_smiling_face:

Reply:
I pushed them to the wrong branch. Should be good now. :sweat_smile: